deployment/postgres-init-db/sql/views/ui/derivation_group_comp.sql

```diff
@@ -0,0 +1,35 @@
+create view ui.derivation_group_comp
```

Comment:
Spoke with @duranb and @AaronPlave about this view and I would _strongly_ prefer if this view was removed in favor of having the UI properly subscribe to this data directly. For example, the following query will net you all but the `derived_total` column (which tbh I don't understand the value of):

```gql
query DGComp {
  derivation_group {
    name
    owner
    source_type_name
    external_sources {
      key
      external_events_aggregate {
        aggregate {
          count
        }
      }
    }
  }
}
```
^ The provided query also removes the duplicated `derivation_group_name` that the current view copies into every entry for some reason

Not only is **not** using a view more performant inherently, but this specific view depends on another view, making its performance even worse.

Reply:
I'm not opposed to using a Hasura query; the idea when we made the view was that the query would be bringing down a lot of potentially unnecessary data and then transforming it, which could incur a lot of network latency (especially as there are more events and sources), whereas the view does any and all transformations upfront. That being said, if a view is that expensive it could be factored out. That being said - the derived total does have its uses in the UI; are you familiar with a way to extract information like that from Hasura? Is that even possible?

Reply:
Yeah, using the `aggregate` as I did here ^. I primarily removed it because I was trying to avoid a second subscription to the `derived_events` view, but you could add a relationship from `external_sources` to `derived_events` (you have the reverse relationship set up) and add that aggregate to the query inside the `external_sources` block in the same way that the `external_events_aggregate` count is handled.

However, I strongly recommend chatting with @duranb and @AaronPlave regarding if the UI already has the info needed to make `derived_total` at the time that you want it--working on data you already have on hand is almost definitely going to be faster than asking for it again via a network request.

Reply:
Completed in the backend in [make derived events a materialized view](https://github.com/NASA-AMMOS/aerie/commit/5b58cd11e16629a1036f1c3a6b59bcaf2b486640) and [factor out derivation_group_comp view](https://github.com/NASA-AMMOS/aerie/commit/c98a7d59510b0813715d518d51eebf528e95cbcb), and in the frontend in [factor out derivation_group_comp view in frontend](https://github.com/NASA-AMMOS/aerie-ui/commit/7373d6d6c4f44c9d84a014d1901ddfe901caf9c9).